include/zephyr/lorawan/lorawan.h

```diff
@@ -378,6 +386,20 @@ int lorawan_set_region(enum lorawan_region region);
  */
 int lorawan_set_channel_mask(enum lorawan_channel_mask mask, bool set_default);
 
+/**
+ * @brief Configure lorawan Rx timeout windows
+ *
+ * Configure one of the receive window timeouts used by the hardware before an
+ * RxTimeout is generated. The particular units of the given timeout depends on
+ * the particular window that is being configured. In general, a value of 0 will
+ * be equivalent to no timeout.
```

Comment:
For passing value of 0 -> no timeout, is this handled somewhere else by lora?

Reply:
That's a part of the loramac stack library yea.